doc-content/drools-docs/src/main/asciidoc/DecisionEngine/engine-event-listeners-development-con.adoc

```diff
@@ -0,0 +1,12 @@
+[id='engine-event-listeners-development-con_{context}']
+= Practices for development of event listeners
+
+The {DECISION_ENGINE} calls event listeners during rule processing. The calls block the execution of the {DECISION_ENGINE}. Therefore, the event listener can affect the  performance of the {DECISION_ENGINE}.
+
+To ensure minimal disruption, observe the following practices:
```

Comment:
```suggestion
To ensure minimal disruption, consider the following practices:
```

Reply:
I would say:
To ensure minimal disruption, observe the following guidelines:
or
To ensure minimal disruption, follow these guidelines: